solution/ui/regulations/package.json

```diff
@@ -5,15 +5,13 @@
     "css": "npx sass -I node_modules css/scss/main.scss ../../static-assets/regulations/css/main.css",
     "css-deploy": "npx sass -I node_modules css/scss/main-deploy.scss ../../static-assets/regulations/css/main.css",
     "watch-css": "npx sass --watch -I node_modules css/scss/main.scss ../../static-assets/regulations/css/main.css",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
-    "watch-js": "npx rollup -c --watch --sourcemap",
-    "js": "npx rollup -c --sourcemap"
+    "storybook": "NODE_OPTIONS=--openssl-legacy-provider start-storybook -p 6006",
+    "build-storybook": "NODE_OPTIONS=--openssl-legacy-provider build-storybook"
```

Comment:
see: https://www.newline.co/@kchan/how-to-fix-the-error-errorerror0308010cdigital-envelope-routinesunsupported--0f8d3f17